test/build/50_ch-image.bats

```diff
@@ -319,6 +319,40 @@ EOF
    [[ $output = *"$CH_IMAGE_STORAGE not a builder storage"* ]]
 }
 
+@test 'ch-image rename' {
+   export CH_IMAGE_STORAGE="$BATS_TMPDIR"/rename
```

Comment:
I think rather than a new pull, you could just restore it at the end, to save downloading again.